### PR TITLE
[Docs] Improved coping UX

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

Recently we have added the possibility to copy code from our documentation. Removing `$` from command line examples is annoying as hell.

### CODE REVIEW

File: docs/book/products/multi_source_inventory.rst [hunk 1]
```diff
@@ -84,7 +84,7 @@ Sylius Plus provides a route that allows changing inventory source stock with an
 
 .. code-block:: bash
 
-   $ POST /api/v1/inventory-sources/{code}/update-stocks
+   POST /api/v1/inventory-sources/{code}/update-stocks
```
Comment (lchrusciel) [inline docs/book/products/multi_source_inventory.rst]: Should be updated and `curl` should be used

File: docs/components_and_bundles/bundles/SyliusTaxationBundle/installation.rst~ [hunk 2]
```diff
@@ -1,143 +0,0 @@
-Installation
```
Comment (lchrusciel) [inline docs/components_and_bundles/bundles/SyliusTaxationBundle/installation.rst~]: This file is wrong and it is ok, that it is removed

**Comment by pamil** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "improve-docs"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Comment by pamil** [comment]
Thank you, Łukasz! :tada:

**Pull request merged**